index.js

```diff
@@ -164,7 +164,17 @@ walletRouter.get('/addresses', (req, res) => {
 walletRouter.get('/tx-history', (req, res) => {
   // TODO Add pagination
   const wallet = req.wallet;
-  res.send(wallet.getTxHistory());
+  const limit = req.query.limit || null;
+  const history = wallet.getTxHistory();
+  if (limit) {
+    const values = Object.values(history);
+    const sortedValues = values.sort((a, b) => b.timestamp - a.timestamp);
+    // XXX The default return is an object and with limit we are changing to an array
+    // Should we keep like this for now? Or maybe change the default return to an array as well?
```

Comment:
I'd change to an array as well, to make sure things remain compatible.

Reply:
Done!